sql/src/main/java/com/vmware/ddlog/DDlogJooqProvider.java

```diff
@@ -89,13 +110,42 @@ private MockResult executeSelectStar(final String sql) throws SQLException {
         return new MockResult(1, result);
     }
 
+    private MockResult executeInsert(final String sql) throws SQLException {
+        final String[] s = sql.replaceAll("[(),]", "")
+                              .split("[ \t\n]+");
+        if (!(s.length >= 4 && s[1].equalsIgnoreCase("INTO")
+                && s[3].equalsIgnoreCase("VALUES"))) {
+            throw new SQLException("Statement not supported: " + sql);
+        }
+        final String tableName = s[2];
+        final int valuesIndex = 4; // the index representing the 1st field of the value being inserted
+        final List<Field<?>> fields = tables.get(tableName.toUpperCase());
+        if (fields == null) {
+            throw new SQLException("Table cannot be queried: " + tableName);
+        }
+        String[] valuesTuple = Arrays.copyOfRange(s, valuesIndex, s.length);
+        final DDlogRecord dDlogRecord = toDDlogRecord(tableName, valuesTuple);
+        final int tableId = dDlogAPI.getTableId("R" + tableName.toLowerCase());
```

Comment:
maybe it's time to create a helper function to return a relation name from a table name.

Reply:
Ack. Will do.